Formula/v8@5.8.rb

```diff
@@ -0,0 +1,118 @@
+# Track Chrome stable.
+# https://omahaproxy.appspot.com/
+class V8AT58 < Formula
+  desc "Google's JavaScript engine"
+  homepage "https://github.com/v8/v8/wiki"
+  url "https://chromium.googlesource.com/chromium/tools/depot_tools.git"
```

Comment:
Should I use the form `url "https://chromium.googlesource.com/v8/v8.git", :tag => "5.8.60"`?